app/views/supervisor/courses/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, t(".title")) %>
+<section class="courses_area p_120">
+  <div class="container">
+    <h3 class="mb-30 title_color"><%= t "courses.supervisor.new.heading" %></h3>
+    <%= form_for [:supervisor, @course] do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+      <div class="form-group">
+        <%= f.label :name, t(".name_course") %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :time, t("courses.supervisor.new.time_estimate") %>
+        <%= f.text_field :time, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :status, t("courses.supervisor.new.status_coures") %>
+        <%= f.text_field :status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label t("courses.supervisor.new.subject")%>
+        <%= f.collection_select(:subject_ids, Subject.all , :id, :name,
```

Comment:
`Subject.all , :id, :name` -> tạo 1 method trong helper mà dùng, ko truy vấn ngoài view
và tìm hiểu về `pluck` để chỉ định các field cần thiết thay vì `.all` 

và thêm scope sắp xếp the tên lun